evm/vm/forks/eip160/constants.py

```diff
@@ -0,0 +1,3 @@
+# https://github.com/ethereum/EIPs/issues/160
+GAS_EXP_EIP160 = 10
+GAS_EXPBYTE_EIP160 = 50
```

Comment:
instead of `forks.eip160` this should be in `forks.byzantium` (since this is actually part of byzantium's fork rules)

Reply:
@pipermerriam When I try to consolidate the constants for `eip160` and `eip170` in `byzantium.constants`, it causes the `SpuriousDragonVM` to throw an error. I suspect it's a circular dependency between SD and Byzantium, which both call each other in `__init__`. What if we did a `forks.constants` module and put all the fork-specific constants there?

Reply:
This is because I'm dumb and told you to move it to byzantium when both this and the eip170 are spurious dragon.  I just tested and if you move them to `forks.spurious_dragon.constants` everything works fine.

Reply:
Haha ok, awesome. I didn't even think to check that, I just launched right into the python/rabbit hole.

Reply:
The clue for me was that an old fork was importing from a newer fork and that should really *never ever* happen.

Reply:
Nice, I'll keep an eye out for that in the future.